src/node_types.cc

```diff
@@ -0,0 +1,69 @@
+#include "node_internals.h"
+
+using v8::Context;
+using v8::FunctionCallbackInfo;
+using v8::Local;
+using v8::Object;
+using v8::Value;
+
+namespace node {
+namespace {
+
+#define VALUE_METHOD_MAP(V)                                                   \
+  V(External)                                                                 \
```

Comment:
Four-space indentation. Ditto elsewhere.

Reply:
@TimothyGu All other macro lists I can find use 2 spaces for indentation?

Reply:
Heh. Never mind then, I guess.